### PR TITLE
Add new North Carolina OCD IDs from NC BOE and BIP (Ballot Info Project)

### PR DESCRIPTION
**Pull request by ccongchen**
Please see a list of new OCD IDs for the general election 2016. The OCD IDs are sourced from the State of North Carolina as well as BIP. Both will be providing election data feeds using the OCD IDs submitted here. Greatly appreciate your timely review and approval. Thanks James!

### CODE REVIEW

**Comment by ccongchen** [comment]
The set of new OCD IDs are for electoral districts in the state of North Carolina.

I have not submitted the new school districts yet as a lot of the new school districts are numbered, and it is not completely straight forward to tell if these are coterminous with the existing ones that contains the actual name in the OCD ID. Any of your feedback is greatly appreciated.

e.g. ocd-division/country:us/state:nc/county:iredell/school_district:2, could it be the same as one of the existing districts in the county of iredell?

ocd-division/country:us/state:nc/county:iredell/school_district:american_renaissance_middle,49B,3700086
ocd-division/country:us/state:nc/county:iredell/school_district:iredell-statesville_schools,490,3702310
ocd-division/country:us/state:nc/county:iredell/school_district:mooresville_city_schools,491,3703120
ocd-division/country:us/state:nc/county:iredell/school_district:pine_lake_preparatory,49,3700145
ocd-division/country:us/state:nc/county:iredell/school_district:success_institute_charter,49D,3700106

Also, can I submit the new school districts in us_school_districts.csv when I don't have the sch_dist_stateid and census_geoid?

File: identifiers/country-us.csv [hunk 1]
```diff
@@ -99164,6 +99164,7 @@ ocd-division/country:us/state:nc/county:caldwell/precinct:pr7-globe~johns_river~
 ocd-division/country:us/state:nc/county:caldwell/precinct:pr8-kings_creek,Caldwell County Precinct Pr8-Kings Creek,,,,,,,,,,
 ocd-division/country:us/state:nc/county:caldwell/school_district:caldwell_county_schools,caldwell county schools,,,,3700580,,,,,140,
 ocd-division/country:us/state:nc/county:camden,Camden County,,,,place-37029,,,,,,
+ocd-division/country:us/state:nc/county:camden/council_district:courthouse_district,Camden County Board Of County Commissioners Courthouse Districtg,,,,,,,,,,
```
Comment (jpmckinney) [inline identifiers/country-us.csv]: "Districtg" typo

File: identifiers/country-us.csv [hunk 2]
```diff
@@ -99294,6 +99299,8 @@ ocd-division/country:us/state:nc/county:cherokee,Cherokee County,,,,place-37039,
 ocd-division/country:us/state:nc/county:cherokee/council_district:1,Cherokee County District 1,,,,,,,,,,
 ocd-division/country:us/state:nc/county:cherokee/council_district:2,Cherokee County District 2,,,,,,,,,,
 ocd-division/country:us/state:nc/county:cherokee/council_district:3,Cherokee County District 3,,,,,,,,,,
+ocd-division/country:us/state:nc/county:cherokee/council_district:4,Cherokee County Council District 4,,,,,,,,,,
```
Comment (jpmckinney) [inline identifiers/country-us.csv]: Name is inconsistent with existing names.

File: identifiers/country-us.csv [hunk 3]
```diff
@@ -99869,6 +99881,10 @@ ocd-division/country:us/state:nc/county:franklin/precinct:9-cedar_rock,Franklin
 ocd-division/country:us/state:nc/county:franklin/school_district:crosscreek_charter,crosscreek charter,,,,3700119,,,,,35A,
 ocd-division/country:us/state:nc/county:franklin/school_district:franklin_county_schools,franklin county schools,,,,3701530,,,,,350,
 ocd-division/country:us/state:nc/county:gaston,Gaston County,,,,place-37071,,,,,,
+ocd-division/country:us/state:nc/county:gaston/council_district:cherryville_township,Gaston County Board Of Commissioners Cherryville Townshipg,,,,,,,,,,
```
Comment (jpmckinney) [inline identifiers/country-us.csv]: More typos with 'g' at the end of the name

File: identifiers/country-us/census_autogenerated/us_census_places.csv [hunk 4]
```diff
@@ -39240,3 +39240,7 @@ ocd-division/country:us/territory:pr/municipio:vieques,Vieques Municipio,place-7
 ocd-division/country:us/territory:pr/municipio:villalba,Villalba Municipio,place-72149
 ocd-division/country:us/territory:pr/municipio:yabucoa,Yabucoa Municipio,place-72151
 ocd-division/country:us/territory:pr/municipio:yauco,Yauco Municipio,place-72153
+ocd-division/country:us/state:nc/county:hyde/place:engelhard,Engelhard unincorporated fishing community,
```
Comment (jpmckinney) [inline identifiers/country-us/census_autogenerated/us_census_places.csv]: This is an auto-generated file and shouldn't be modified if you want those modifications to persist.

**Comment by ccongchen** [comment]
Thanks James for your feedback! Will fix the issues. Any comments on how to handle the school districts with numbers only (whereas the existing values contain school district names)? I'm inclined to submit them as-is and link to the existing values as aliases once we get additional info on these school districts. What do you think?  I'm reaching out to NC BOE to get their feedback.

**Comment by jpmckinney** [comment]
If they are identical, then either a `sameAs` column should be added to refer to the canonical ID, or a new file should be created to establish the `sameAs` relationship.

**Comment by ccongchen** [comment]
PTAL. Thanks James!

File: identifiers/country-us/state-nc-local_gov.csv [hunk 5]
```diff
@@ -260,3 +260,68 @@ ocd-division/country:us/state:nc/place:cary/council_district:a,Cary NC city coun
...
+ocd-division/country:us/state:nc/county:hyde/council_district:lake_landing_township,Hyde County Board Of Commissioners Lake Landing Township
+ocd-division/country:us/state:nc/county:hyde/council_district:ocracoke_township,Hyde County Board Of Commissioners Ocracoke Township
+ocd-division/country:us/state:nc/county:johnston/place:benson/council_district:1,Town Of Benson Commissioner District 1
+ocd-division/country:us/state:nc/county:johnston/place:benson/council_district:2,Town Of Benson Commissioner District 2
+ocd-division/country:us/state:nc/county:martin/council_district:eastern,Martin County Commissioner Eastern District
+ocd-division/country:us/state:nc/county:montgomery/council_district:1,Montgomery County Board Of Commissioners District 1
+ocd-division/country:us/state:nc/county:montgomery/council_district:2,Montgomery County Board Of Commissioners District 2
+ocd-division/country:us/state:nc/county:montgomery/council_district:3,Montgomery County Board Of Commissioners District 3
+ocd-division/country:us/state:nc/county:pamlico/council_district:1,Pamlico County Board Of  Commissioners District 1
```
Comment (jpmckinney) [inline identifiers/country-us/state-nc-local_gov.csv]: There shouldn't be two spaces in these names.

File: identifiers/country-us/census_autogenerated/us_census_places.csv (hunk 4 above)
Comment (jpmckinney) [inline identifiers/country-us/census_autogenerated/us_census_places.csv]: You still shouldn't be editing this file at all.

File: identifiers/country-us/state-nc-local_gov.csv [hunk 6]
```diff
@@ -27,9 +27,9 @@ ocd-division/country:us/state:nc/county:chatham/council_district:2,Chatham Count
 ocd-division/country:us/state:nc/county:chatham/council_district:3,Chatham County District 3
 ocd-division/country:us/state:nc/county:chatham/council_district:4,Chatham County District 4
 ocd-division/country:us/state:nc/county:chatham/council_district:5,Chatham County District 5
-ocd-division/country:us/state:nc/county:cherokee/council_district:1,Cherokee County District 1
-ocd-division/country:us/state:nc/county:cherokee/council_district:2,Cherokee County District 2
-ocd-division/country:us/state:nc/county:cherokee/council_district:3,Cherokee County District 3
+ocd-division/country:us/state:nc/county:cherokee/council_district:1,Cherokee County Council District 1
```
Comment (jpmckinney) [inline identifiers/country-us/state-nc-local_gov.csv]: I don't understand why the word 'Council' needs to be added. It's not consistent with the names of existing council districts.

File: identifiers/country-us.csv (hunk 1 above)
Comment (ccongchen) [inline identifiers/country-us.csv]: Fixed.

File: identifiers/country-us.csv (hunk 3 above)
Comment (ccongchen) [inline identifiers/country-us.csv]: Fixed.

File: identifiers/country-us/census_autogenerated/us_census_places.csv (hunk 4 above)
Comment (ccongchen) [inline identifiers/country-us/census_autogenerated/us_census_places.csv]: Ack. Fixed.

File: identifiers/country-us/state-nc-local_gov.csv (hunk 6 above)
Comment (ccongchen) [inline identifiers/country-us/state-nc-local_gov.csv]: Fixed.

File: identifiers/country-us/state-nc-local_gov.csv (hunk 5 above)
Comment (ccongchen) [inline identifiers/country-us/state-nc-local_gov.csv]: Fixed.

**Comment by ccongchen** [comment]
Thanks James for your timely feedback. PTAL.

File: identifiers/country-us/census_autogenerated/us_census_places.csv (hunk 4 above)
Comment (jpmckinney) [inline identifiers/country-us/census_autogenerated/us_census_places.csv]: There are still 3 changed lines in this file. The file shouldn't be changed at all.

File: identifiers/country-us/us_school_districts.csv [hunk 7]
```diff
@@ -17176,3 +17176,54 @@ ocd-division/country:us/state:wy/county:washakie/school_district:washakie_county
...
+ocd-division/country:us/state:nc/county:pitt/school_district:6,Pitt County School District 6
+ocd-division/country:us/state:nc/county:pitt/school_district:7,Pitt County School District 7
+ocd-division/country:us/state:nc/county:pitt/school_district:8,Pitt County School District 8
+ocd-division/country:us/state:nc/county:pitt/school_district:9,Pitt County School District 9
+ocd-division/country:us/state:nc/county:vance/school_district:3,Vance County School District 3
+ocd-division/country:us/state:nc/county:vance/school_district:4,Vance County School District 4
+ocd-division/country:us/state:nc/county:vance/school_district:5,Vance County School District 5
+ocd-division/country:us/state:nc/county:washington/school_district:1,Washington County School District 1
+ocd-division/country:us/state:nc/county:wayne/school_district:2,Wayne County School District 2
+ocd-division/country:us/state:nc/county:surry/place:elkin/school_district:city,Elkin_City_Schools_School_District District City
```
Comment (jpmckinney) [inline identifiers/country-us/us_school_districts.csv]: This (and some other) names have underscores and don't make sense.

File: identifiers/country-us/state-nc-local_gov.csv [hunk 8]
```diff
@@ -260,3 +260,181 @@ ocd-division/country:us/state:nc/place:cary/council_district:a,Cary NC city coun
...
+ocd-division/country:us/state:nc/county:wake/school_board_district:8,Wake County School Board District 8
+ocd-division/country:us/state:nc/county:wake/school_board_district:9,Wake County School Board District 9
+ocd-division/country:us/state:nc/county:warren/school_board_district:3,Warren County Board Of Education-district 3
+ocd-division/country:us/state:nc/county:warren/school_board_district:5,Warren County Board Of Education-district 5
+ocd-division/country:us/state:nc/county:washington/school_board_district:3,Washington County Board Of Education District 3
+ocd-division/country:us/state:nc/county:wayne/school_board_district:3,Wayne County School Board District 3
+ocd-division/country:us/state:nc/county:wilson/school_board_district:2,Wilson County School Board District 2
+ocd-division/country:us/state:nc/county:wilson/school_board_district:4,Wilson County School Board District 4
+ocd-division/country:us/state:nc/county:wilson/school_board_district:6,Wilson County School Board District 6
+ocd-division/country:us/state:nc/county:cabarrus/place:kannapolis,Kannapolis city
```
Comment (jpmckinney) [inline identifiers/country-us/state-nc-local_gov.csv]: This (and others) has unusual capitalization.

File: identifiers/country-us/state-nc-local_gov.csv [hunk 9]
```diff
@@ -260,3 +260,181 @@ ocd-division/country:us/state:nc/place:cary/council_district:a,Cary NC city coun
...
+ocd-division/country:us/state:nc/county:wake/school_board_district:1,Wake County School Board District 1
+ocd-division/country:us/state:nc/county:wake/school_board_district:2,Wake County School Board District 2
+ocd-division/country:us/state:nc/county:wake/school_board_district:3,Wake County School Board District 3
+ocd-division/country:us/state:nc/county:wake/school_board_district:4,Wake County School Board District 4
+ocd-division/country:us/state:nc/county:wake/school_board_district:5,Wake County School Board District 5
+ocd-division/country:us/state:nc/county:wake/school_board_district:6,Wake County School Board District 6
+ocd-division/country:us/state:nc/county:wake/school_board_district:7,Wake County School Board District 7
+ocd-division/country:us/state:nc/county:wake/school_board_district:8,Wake County School Board District 8
+ocd-division/country:us/state:nc/county:wake/school_board_district:9,Wake County School Board District 9
+ocd-division/country:us/state:nc/county:warren/school_board_district:3,Warren County Board Of Education-district 3
```
Comment (jpmckinney) [inline identifiers/country-us/state-nc-local_gov.csv]: This (and others) has unusual capitalization and an inexplicable hyphen.

File: identifiers/country-us/state-nc-local_gov.csv [hunk 10]
```diff
@@ -260,3 +260,181 @@ ocd-division/country:us/state:nc/place:cary/council_district:a,Cary NC city coun
...
+ocd-division/country:us/state:nc/county:pender/school_board_district:5,Pender County Board Of Education District 5
+ocd-division/country:us/state:nc/county:polk/school_board_district:saluda_township,Polk County Board Of Education Saluda Township
+ocd-division/country:us/state:nc/county:polk/school_board_district:tryon_township,Polk County Board Of Education Tryon Township
+ocd-division/country:us/state:nc/county:polk/school_board_district:white_oak_township,Polk County Board Of Education White Oak Township
+ocd-division/country:us/state:nc/county:rockingham/school_board_district:1,Rockingham County School Board District 1
+ocd-division/country:us/state:nc/county:rockingham/school_board_district:2,Rockingham County School Board District 2
+ocd-division/country:us/state:nc/county:rockingham/school_board_district:3,Rockingham County School Board District 3
+ocd-division/country:us/state:nc/county:rockingham/school_board_district:4,Rockingham County School Board District 4
+ocd-division/country:us/state:nc/county:rowan/school_board_district:rowan-salisbury_3,Rowan-salisbury Board Of Education Seat 3
```
Comment (jpmckinney) [inline identifiers/country-us/state-nc-local_gov.csv]: In the name, Salisbury (here and elsewhere) should be capitalized.

File: identifiers/country-us/state-nc-local_gov.csv [hunk 11]
```diff
@@ -260,3 +260,181 @@ ocd-division/country:us/state:nc/place:cary/council_district:a,Cary NC city coun
...
+ocd-division/country:us/state:nc/county:bladen/school_board_district:1,Bladen County School Board District 1
+ocd-division/country:us/state:nc/county:bladen/school_board_district:2,Bladen County School Board District 2
+ocd-division/country:us/state:nc/county:brunswick/school_board_district:1,Brunswick County Board Of Education District 1
+ocd-division/country:us/state:nc/county:brunswick/school_board_district:2,Brunswick County Board Of Education District 2
+ocd-division/country:us/state:nc/county:brunswick/school_board_district:4,Brunswick County Board Of Education District 4
+ocd-division/country:us/state:nc/county:buncombe/school_board_district:north_buncombe,Buncombe County School Board North_Buncombe District
+ocd-division/country:us/state:nc/county:buncombe/school_board_district:owen,Buncombe County School Board Owen District
+ocd-division/country:us/state:nc/county:buncombe/school_board_district:roberson,Buncombe County School Board Roberson District
+ocd-division/country:us/state:nc/county:cabarrus/place:kannapolis/school_board_district:1,Kannapolis Board Of Education- Area I
```
Comment (jpmckinney) [inline identifiers/country-us/state-nc-local_gov.csv]: This (and others) has an inexplicable hyphen.

File: identifiers/country-us/us_school_districts.csv (hunk 7 above)
Comment (ccongchen) [inline identifiers/country-us/us_school_districts.csv]: Fixed.

File: identifiers/country-us/state-nc-local_gov.csv (hunk 11 above)
Comment (ccongchen) [inline identifiers/country-us/state-nc-local_gov.csv]: Removed all inexplicable hyphens.

File: identifiers/country-us/state-nc-local_gov.csv (hunk 10 above)
Comment (ccongchen) [inline identifiers/country-us/state-nc-local_gov.csv]: Done.

File: identifiers/country-us/state-nc-local_gov.csv (hunk 9 above)
Comment (ccongchen) [inline identifiers/country-us/state-nc-local_gov.csv]: Removed.

File: identifiers/country-us/state-nc-local_gov.csv (hunk 8 above)
Comment (ccongchen) [inline identifiers/country-us/state-nc-local_gov.csv]: This is the same capitalization as in the auto-generated census places file. Once I discovered that, it seems to me that we shall not add this as a new OCD ID (under cabarrus county), so I removed it.

**Comment by ccongchen** [comment]
All issues fixed. PTAL. Thanks James!

**Comment by ccongchen** [comment]
Hi James, appreciate if you could review and let us know your feedback. We're hoping to get this finalized as soon as we can. Thank you!

**Comment by ccongchen** [comment]
Hi James, quick ping on this. Please let us know what you think! I'd like to submit the changes as quickly as possible. Note that the elections team at Google has a number of upcoming launches that can benefit from the added OCD IDs.

**Comment by jpmckinney** [comment]
@ccongchen I still have concerns about the data quality. I'm asking @jamesturk and @paultag to review.

**Comment by ccongchen** [comment]
SG. Thanks James! Really appreciate your help and I will try my best to address any concerns you/other reviewers may have.

**Comment by jamesturk** [comment]
Just had a chance to look this over this morning actually, I don't see any issues.  Some of the numbered school districts seem weird but I don't know how the numbering works and if the missing numbers exist I assume they can be added later.

**Comment by ccongchen** [comment]
Thanks a lot! @jamesturk

**Pull request merged**